src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -170,7 +170,9 @@ const VitalDetails = props => {
 
   useEffect(
     () => {
-      if (currentPage > 1 && records?.length) {
+      if (currentPage === 1) {
```

Comment:
Since the h1 is within a block that only displays if there are records, you shouldn't need this extra block, in fact, it might break the code since it could try to focus an h1 when there is none displayed.

Reply:
fixed